src/data/miscLunch.json

```diff
@@ -0,0 +1,69 @@
+{
+    "menuGroups": [
+        {
+            "Grill": [
+                "Hamburger",
+                "Cheeseburger",
+                "Chicken Tenders",
+                "Black Bean Burger",
+                "Turkey Burger",
+                "Fries"
+            ],
+            "Pizza": [
+                "Cheese",
+                "Pepperoni",
+                "Sausage",
+                "Specialty Pizza",
+                "Cheese Bread Sticks"
+            ]
+        },
+        {
+            "PWC Jazzmans": [
+                "Acai Bowls",
+                "Salads",
+                "Grains",
+                "Wraps",
+                "Sandwitches",
+                "Fruit Cup",
+                "Yogurt",
+                "Antipasto Box",
+                "Protein Box",
+                "Bubble Tea"
+            ],
+            "Coffee and Tea": [
```

Comment:
It seems that "sides "and "beverages" accompany cafeteria food, while PWC Jazzmans, Grill and Pizza is a separate location. what is "Coffee and Tea"? is that a location of where to get food? 

Is there some way we can make it more clear where these items are? I'm thinking some location tag and icon that's like these items are here. Thoughts on this?

Reply:
The arrangement ive used is just grouped by order on the school website, but the groups correspond roughly with location. Beverages and Sides are located in the same place as the cafeteria. The grill and pizza is in the wood commons, and the 'Coffee and Tea' category is at Jazzman's. I moved the beverages and sides to the top of the list because it makes more sense in proximity to the rest of the cafeteria foods, but otherwise I think the arrangement makes sense. The locations are common knowledge in my experience, so I'm not sure how useful a location tag would be